src/chain_spec.rs

```diff
@@ -876,6 +882,19 @@ fn centrifuge_genesis(
 		},
 		treasury: Default::default(),
 		interest_accrual: Default::default(),
+		block_rewards: centrifuge_runtime::BlockRewardsConfig {
+			collators: initial_authorities
+				.iter()
+				.cloned()
+				.map(|(acc, _)| acc)
+				.collect(),
+			collator_reward: 6_000 * AIR,
```

Comment:
Should `AIR` here be `CFG`?  It looks like AIR is set to `CFG` in altair consts, but wondering if it would be better/clearer to use `CFG` given that this is for cent runtime.   Not blocking at all, just curious :).

Reply:
Great catch! Absolutely. Thanks!